### ISSUE TITLE
`getDevices` command seems to not work anymore

### ISSUE THREAD

**Issue by arthur-fontaine** [okwasniewski/MiniSim#119]
The command `getDevices` that can be used with AppleScript and that is used by the Raycast extension make MiniSim crash.

I use the version 0.8.3, and I am on a MacBook Pro M1 Pro.

The AppleScript:
```
tell application "MiniSim"
    getDevices platform "android"
end tell
```

**Comment by okwasniewski** [comment]
Thanks for reporting the issue, should be fixed in 0.8.4

**Comment by arthur-fontaine** [comment]
Can confirm it works now. Thanks you're a goat
